api/index.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
+const cors = require('cors');
```

Comment:
is this dependency included in the project?

Reply:
Do you refer to cors?, If so,  I think we can delete it.

Reply:
yes, cors, you can add it if you need it